From patchwork Thu Apr 22 04:39:56 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Stephen Rothwell <sfr@canb.auug.org.au>
X-Patchwork-Id: 12217627
Return-Path: <kvm-owner@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-10.8 required=3.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_PATCH,
	MAILING_LIST_MULTI,SPF_HELO_NONE,SPF_PASS autolearn=unavailable
	autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id A32ACC433B4
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 04:40:05 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 78F9561450
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 04:40:05 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S229636AbhDVEkg (ORCPT <rfc822;kvm@archiver.kernel.org>);
        Thu, 22 Apr 2021 00:40:36 -0400
Received: from ozlabs.org ([203.11.71.1]:53873 "EHLO ozlabs.org"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S229441AbhDVEkf (ORCPT <rfc822;kvm@vger.kernel.org>);
        Thu, 22 Apr 2021 00:40:35 -0400
Received: from authenticated.ozlabs.org (localhost [127.0.0.1])
        (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
         key-exchange ECDHE (P-256) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
        (No client certificate requested)
        by mail.ozlabs.org (Postfix) with ESMTPSA id 4FQl8G53X5z9sRf;
        Thu, 22 Apr 2021 14:39:57 +1000 (AEST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=canb.auug.org.au;
        s=201702; t=1619066399;
        bh=gytcSQ0EK1A2/+Z/7YRsitFG89wvMlfVFxtZpQz7+Do=;
        h=Date:From:To:Cc:Subject:From;
        b=oi7ot3VlB0LJnu8c+jxixbdFRQPWaz1M5SDl2JQdE6yWZTeU4ngUXMbY1G1GXbzZ0
         SQtVxrOABbsDhhC0BsD0UoZG5jmkrx4zEevSHELyHHekwAuvrvF5h+KtTvvnHtufU/
         fH3ayb9bbe0MVRzvOgQFKbKzqbRxy4YDu7mq+hgFtptml9z7fmIiMMNcw0zduLRKqT
         RkDdUB3rwpSnspYdDR9+Bep4FzV8d6uU3ruK0crjkQRkJPwPx9Xzx1PkNsBL1azoTz
         eVO0NGPL4NjJltfxgUxjkSYq+YytjeSTA2KDbTxlEL4KtsobQDWp++k6wEKyU4szMd
         PRNBAQstqPvFg==
Date: Thu, 22 Apr 2021 14:39:56 +1000
From: Stephen Rothwell <sfr@canb.auug.org.au>
To: Christoffer Dall <cdall@cs.columbia.edu>,
        Marc Zyngier <maz@kernel.org>,
        Paolo Bonzini <pbonzini@redhat.com>, KVM <kvm@vger.kernel.org>
Cc: Emanuele Giuseppe Esposito <eesposit@redhat.com>,
        Jianyong Wu <jianyong.wu@arm.com>,
        Linux Kernel Mailing List <linux-kernel@vger.kernel.org>,
        Linux Next Mailing List <linux-next@vger.kernel.org>
Subject: linux-next: manual merge of the kvm-arm tree with the kvm tree
Message-ID: <20210422143956.5a43d00a@canb.auug.org.au>
MIME-Version: 1.0
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Hi all,

Today's linux-next merge of the kvm-arm tree got a conflict in:

  Documentation/virt/kvm/api.rst

between commit:

  24e7475f931a ("doc/virt/kvm: move KVM_CAP_PPC_MULTITCE in section 8")

from the kvm tree and commit:

  3bf725699bf6 ("KVM: arm64: Add support for the KVM PTP service")

from the kvm-arm tree.

I fixed it up (see below) and can carry the fix as necessary. This
is now fixed as far as linux-next is concerned, but any non trivial
conflicts should be mentioned to your upstream maintainer when your tree
is submitted for merging.  You may also want to consider cooperating
with the maintainer of the conflicting tree to minimise any particularly
complex conflicts.

diff --cc Documentation/virt/kvm/api.rst
index ee3446bf2a01,3f210953c135..000000000000
--- a/Documentation/virt/kvm/api.rst
